src/extension.ts

```diff
@@ -7,22 +7,32 @@ const SpotifyWebApi = require('spotify-web-api-node');
 import { RecentProvider } from './recent';
 import { clientId, clientSecret } from './secrets';
 
+import {init} from './statusBar'
+
 // Auth config
 const PATH = '/spotify-callback';
 const PORT = 8350;
 
+
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
+
+
 	let spotify = new SpotifyWebApi({
 		redirectUri: `http://localhost:${PORT}${PATH}`,
 		clientId,
 		clientSecret
 	});
 	spotifyAuthentication(spotify);
 
+    // Initialise status bar stuff
+    init(context)
```

Comment:
I think this init system is super clever and a great way to avoid repeating code, but I'd like this function to be something much more specific in case we want to use similar systems for other modules. I'm not super familiar with JavaScript, but surely there's a way to set this up like `statusBar.init()`?

Reply:
Yeah true, I thought I had changed it as I planned using the term `init` only for testing purposes but forgot to commit.
Will update this asap